src/retrace/retrace.py

```diff
@@ -19,11 +19,11 @@
 from signal import *
 from subprocess import *
 import magic
-from argparser import *
+from .argparser import *
 from webob import Request
 from yum import YumBase
 from config import *
```

Comment:
Missed this one.

Reply:
@mkutlak , thank you for the review. However, I do not understand your comment. May I ask you clarify more?

Reply:
`from config import *` imports config options from [config.py.in](https://github.com/abrt/retrace-server/blob/master/src/retrace/config.py.in), therefore it should be 
`from .config import *`, or am I wrong?

Reply:
@mkutlak I see now, you are right, thanks. I update the patch.